index.js

```diff
@@ -51,6 +51,12 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var outputDir = this.options.path || '.';
+      var outputFilename = path.join(outputDir, this.options.filename || DEFAULT_OUTPUT_FILENAME);
+      var obj = JSON.parse(fs.readFileSync(outputFilename, 'utf8'));
+      if (obj.status == 'done') {
+          chunks = obj.chunks;
+      }
```

Comment:
well, I use another webpack plugin to clear webpack-stats.json before every build, I will figure out a way to auto clear webpack-stats.json instead of using an extra plugin.